app/Models/ConfigurationSetter.php

```diff
@@ -149,7 +149,7 @@ private function _ttl_default(&$data, $value) {
 
 	private function _view_mode(&$data, $value) {
 		$value = strtolower($value);
-		if (!in_array($value, array('global', 'normal', 'reader'))) {
+		if (!in_array($value, array('global', 'normal', 'reader', 'contextview'))) {
```

Comment:
I would better like a name that does not end with -view, to be closer to the other names. `context` might be fine, but maybe we can find something better. What is used in other software proposing such a view? Maybe *preview*, *timeline*, *magazine*...

Reply:
Yeah, I agree.